package.json

```diff
@@ -28,6 +29,7 @@
   },
   "homepage": "https://github.com/alanhoff/node-portastic#readme",
   "dependencies": {
+    "@types/node": "0.0.2",
```

Comment:
Note that you expose an object that subclasses `EventEmitter` from `node` - a consumer of your API therefore needs the node typing information to fully understand your public API, hence the addition of the node types as dependency and not a devDependency (see [_Packaging dependent declarations_](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html))